Serialization.Test/TestSerialization.fs

```diff
@@ -175,4 +175,87 @@ module RecordSerialization =
                 0xD8uy ||| 0uy; 0x02uy; // tag: fldnum=43, varint
                 0x01uy;                 // value true
             |]
+            
+    type Proto3Message  = {
+        id : int32
+        name : string
+        _unknownFields : RawField list
+        }
+        with
+            static member Default = {
+                id = 0
+                name = ""
+                _unknownFields = List.empty
+                }
+
+            static member Serializer (m, zcb) =
+                (m.id            |> Encode.fromVarint 1) >>
+                (m.name          |> Encode.fromString 2)
+                <| zcb
+
+            static member DecoderRing =
+                [
+                    0, fun m rawField -> { m with _unknownFields = rawField :: m._unknownFields } : Proto3Message
+                    1, fun m rawField -> { m with id = rawField |> Decode.toInt32 } : _
+                    2, fun m rawField -> { m with name = rawField |> Decode.toString } : _
+                ]
+                |> Map.ofList
+    
+            static member DecodeFixup m =
+                { m with _unknownFields = List.rev m._unknownFields }
+
+            static member RequiredFields =
+                [ 1; 2 ] |> Set.ofList
+
+            static member UnknownFields m =
```

Comment:
Same as above: should you remove this to show usage without unknown fields?  Or, comment this is optional?